cpp/src/Ice/ObjectAdapterFactory.cpp

```diff
@@ -114,7 +114,7 @@ IceInternal::ObjectAdapterFactory::updateObservers(void (ObjectAdapterI::*fn)())
 
 ObjectAdapterPtr
 IceInternal::ObjectAdapterFactory::createObjectAdapter(
-    string name,
+    string name, // NOLINT(performance-unnecessary-value-param)
```

Comment:
Can we instead move it to OA constructor, and then use getName for inserting in _adapterNamesInUse map

Reply:
It doesn't work well because we use name at the end to remove it from adapterNamesInUse on exception. It's much simpler to ignore this lint.